solana/src/burn.rs

```diff
@@ -1,35 +1,18 @@
-use crate::{send_with_retry, GetSignature, SolanaRpcError};
-use anchor_client::RequestBuilder;
-use async_trait::async_trait;
-use chrono::{DateTime, Utc};
-use helium_anchor_gen::{
-    anchor_lang::{AccountDeserialize, ToAccountMetas},
-    data_credits::{self, accounts, instruction},
-    helium_sub_daos::{self, DaoV0, SubDaoV0},
+use crate::{
+    read_keypair_from_file, sender, GetSignature, Keypair, Pubkey, SolanaRpcError, SubDao,
```

Comment:
these seem to be re-exports from from the helium-lib crate, some of which seem to be re-exports from the solana_sdk crate? personally not a huge fan of re-exports (would rather know right away where to go to see the code if i need to know what it's doing) but if we're going to do that can we be consistent about it?

Reply:
I think ideally, all `solana_sdk` things should come from `helium-lib` and we can drop the solana dependency from this workspace. But that is still a work in progress.